ee/tabby-db/src/password_reset.rs

```diff
@@ -48,9 +57,21 @@ impl DbConn {
         Ok(password_reset)
     }
 
+    pub async fn mark_password_reset_expired(&self, code: &str) -> Result<()> {
```

Comment:
this is only for testing? mark it with `#[cfg(test]`

Reply:
Tried this, but it becomes inaccessible from the webserver crate even in `#[cfg(test)]` blocks when I do this.

Reply:
You probably need https://github.com/TabbyML/tabby/blob/76f733b216b996cec6f5dad9ae69da20d2bd60f8/ee/tabby-db/src/lib.rs#L225